### PR TITLE
Initial .NET documentation

### PR DESCRIPTION
**Pull request by austinlparker**
This is the index and quickstart docs for .NET, so we can get something going there.

### CODE REVIEW

File: content/en/docs/net/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+OpenTelemetry for .NET supports all officially supported versions of [.NET Core](https://dotnet.microsoft.com/download/dotnet-core) and [.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework) except for .NET Framework 3.5 SP1.
+
+# Status and Releases
+
+This is the current release status for OpenTelemetry components in this language:
+
+| Tracing | Metrics | Logging |
+| ------- | ------- | ------- |
+| Beta    | Alpha   | Beta    |
```
Comment (flands) [inline content/en/docs/net/_index.md:16]: Is logging really beta?

**Review by flands — APPROVED** [review]
LGTM -- quick question

File: content/en/docs/net/_index.md (hunk 1 above)
Comment (austinlparker) [reply content/en/docs/net/_index.md:16]: According to their readme, yeah.

**Comment by SergeyKanzhelev** [comment]
CC: @ank3it, @cijothomas

**Pull request merged**